src/database/models.py

```diff
@@ -602,43 +615,47 @@ def full_json(self):
         carbon_footprint_reduction = 0
         for actionRel in done_actions:
             if actionRel.action and actionRel.action.calculator_action:
-                carbon_footprint_reduction += AverageImpact(actionRel.action.calculator_action, actionRel.date_completed)
+                carbon_footprint_reduction += AverageImpact(
+                    actionRel.action.calculator_action, actionRel.date_completed
+                )
 
         goal["organic_attained_carbon_footprint_reduction"] = carbon_footprint_reduction
 
         # calculate values for community impact to be displayed on front-end sites
-        impact_page_settings: ImpactPageSettings = ImpactPageSettings.objects.filter(community__id=self.pk).first()
+        impact_page_settings: ImpactPageSettings = ImpactPageSettings.objects.filter(
```

Comment:
half the time when one of you guys does a reformatting like this, the other one changes it back.  Maybe depends on your mood?